cribbslab/pipeline_pseudobulk.py

```diff
@@ -0,0 +1,132 @@
+"""===========================
+Pseudoalignment pipeline for bulkRNAseq data
+author: Alina Kurjan
+===========================
+To run locally: python pipeline_pseudobulk.py make full --local
```

Comment:
you have added somehting that isnt great I think it should be changed to this..

Reply:
your wrong